app/controllers/spotlight/featured_images_controller.rb

```diff
@@ -0,0 +1,52 @@
+module Spotlight
+  # Handles requests to upload images for exhibit thumbnails
+  class FeaturedImagesController < Spotlight::ApplicationController
+    load_and_authorize_resource instance_name: :featured_image
+
+    def create
+      @featured_image.save!
+      if @featured_image.image.file
+        render json: { tilesource: tilesource, id: @featured_image.id }
+      else
+        render json: { error: 'unable to create image' }, status: :bad_request
+      end
+    end
+
+    private
+
+    def tilesource
+      riiif.info_url(@featured_image.id)
+    end
+
+    # The create action can be called from a number of different forms, so
+    # we normalize all the parameters.
+    # rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength, Metrics/PerceivedComplexity
+    def create_params
```

Comment:
Yeah, I'll see if I can just set a static name on the file input.  We'll need to support at least two keys (since thumbnails and mastheads can exist in the same form, and it would be odd to have two file inputs w/ the same name in them.